### DISCUSSION TITLE
Elektronika SM 7238 can't boot, noob here, any help would be appreciated!

### DISCUSSION THREAD

**Discussion by mrpenguinb** [mamedev/discussions#118] [Q&A]
I've been trying to boot the SM7238 terminal in MAME with no success. Booting it through MAME's GUI only shows "Xaume" blinking.
![image](https://github.com/mamedev/discussions/assets/13060086/9c8772d8-31ee-4c01-9a7d-d5b2578f1757)
I feel like I'm missing something basic here, any guidance would be great.

https://wiki.mamedev.org/index.php/Driver:Soviet_terminals

**Comment by justnibio** [comment]
I don't think it can work independently, it's a terminal, I think it must be connected to a network in some way
